aries-site/src/pages/templates/reverse-navigation.mdx

```diff
@@ -1,3 +1,36 @@
-import { ComingSoon } from '../../components';
+import { Image } from 'grommet';
+import { DiagramPreview, Example } from '../../layouts';
+import {
+  NameValueListFontWeightSecondPreview,
+} from '../../examples';
 
-<ComingSoon />
\ No newline at end of file
+## When to Use:
+
+- When the user may want to return to the previous page’s content
+- When the user may want to trace through their steps chronologically to return to past page content
+- For web applications, as all web browsers contain built in chronological content buttons
+
+## Reverse Navigation Examples:
+
+### In-App Example
+Provide a context button for when it is valuable to provide users explicitly around where they are returning to.
```

Comment:
I had left a few comments on the Figma file re: wording suggestions on this page -- we can wait for Vicky's replies there.

Reply:
sounds good I had left what I had until I had a chance to go over with Vicky